jest/helpers/index.js

```diff
@@ -0,0 +1,26 @@
+export const getRandomElement = arr =>
+  arr[Math.floor(Math.random() * arr.length)]
```

Comment:
must be `arr[Math.floor(Math.random() * (arr.length-1))]`

edge case, `math.random()` is 1 --> trying to access `arr[arr.length]`